unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+include(FetchContent)
+
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest.git
+  GIT_TAG        release-1.10.0
+)
+FetchContent_GetProperties(googletest)
+if(NOT googletest_POPULATED)
+  FetchContent_Populate(googletest)
+  set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+  add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
+endif()
+include(GoogleTest)
+
+include_directories("${gtest_SOURCE_DIR}/include")
+
+ADD_SUBDIRECTORY(noddy)
```

Comment:
Sorry we need to get change Noddy as well...  (It refers to a character created by Enid Blyton).

Reply:
Done